### DISCUSSION TITLE
Contributing to real opensource projects

### DISCUSSION THREAD

**Discussion by ghost** [community/community#22887] [New to GitHub]
<p>Hi guys, I’m new to git and github and I am trying to explore opensource projects but can’t figure out how to contribute real opensource projects.  I am looking for ReactJS and  JavaScript projects.</p>
<p>Please share some advice how can I identify real opensource projects in ReactJS and JavaScrript? </p>
<p>Thanks </p>

**Comment by enderahmetyurt** [comment, marked as answer]
<p>Hey Azambadar,</p>
<p>You can check <a href="https://github.com/explore" rel="nofollow noopener">https://github.com/explore</a>. You can find awesome projects from there. In my opion if a project on Github have many stars like +500, it should be a real open source project. You can filter and sort project from explore page.</p>
<p>Cheers.</p>

**Comment by lee-dohm** [comment]
<p>Open source projects are best when the people who are participating in them are also the people that use the thing that is produced by the project. So, I would first ask, “What open source tools, libraries, or applications do you use that use JavaScript and React?” Do you use <a href="https://desktop.github.com" rel="nofollow noopener">GitHub Desktop</a>? It’s based on React and TypeScript. Do you use some website that is open source and uses those pieces in its infrastructure?</p>
<p>If there’s nothing that you use personally, but you still want to get involved in open source, I would recommend taking a look at <a href="https://www.firsttimersonly.com/" rel="nofollow noopener">https://www.firsttimersonly.com/</a>. It can point you to a number of projects that want to help mentor people new to open source and have set up systems to help them be successful.</p>
<p>Good luck and let us know if you have any questions!</p>
